resources/dicts/events/ceremonies/ceremony_traits.json

```diff
@@ -229,7 +318,12 @@
         "cunning",
         "tranquility",
         "insight",
-        "observance"
+        "observance",
+        "caution",
+        "intellligence",
+        "dignity",
+        "truthfulness",
```

Comment:
might be better replaced with 'integrity'?

Reply:
you mean truthfulness right? sure i can swap that out for integrity!